gnuradio-runtime/python/gnuradio/gr/gateway.py

```diff
@@ -46,7 +46,7 @@ class array_like:
 ########################################################################
 class gateway_handler(gr.feval_ll):
 
-    #dont put a constructor, it wont work
+    #don't put a constructor, it won't work
```

Comment:
This would be a nice chance to improve the wording. "Don't define" would be better, I guess. But I can't see/remember what's going on here, so, do this according to your own best judgment.

Reply:
not sure either, therefore I won't mess with it.